doc/api/worker_threads.md

```diff
@@ -49,6 +49,10 @@ The above example spawns a Worker thread for each `parse()` call. In actual
 practice, use a pool of Workers instead for these kinds of tasks. Otherwise, the
 overhead of creating Workers would likely exceed their benefit.
 
+Worker threads inherit non-process-specific options by default. Refer to
+[`Worker constructor options`][] to know how to customize worker thread options,
+specifically `argv` and `execArgv` options.
```

Comment:
The two paragraphs here belong together; this should be added either above, or (preferably) below them.

Reply:
@addaleax, Thanks. Put it in the suggested place.